src/tools/pub/pubspec.md

```diff
@@ -522,4 +522,11 @@ you must specify a Dart SDK constraint with a minimum version of
 at least 1.19.0, to ensure that older versions of pub won't
 accidentally install packages that need Flutter.
 
+{{site.alert.version-note}}
+  Since Flutter 2.0,
+  enforcing of the Flutter SDK constraint upper bound is disabled. 
+  For more information, check out
+  this [Issue](https://github.com/flutter/flutter/issues/95472).
```

Comment:
**issue:** As Flutter 2.0 falls in the far past, you can omit that detail. Also, write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
  The Flutter SDK disabled the enforcement of the constraint upper bound. 
  To learn more, check out [issue #95472](https://github.com/flutter/flutter/issues/95472)
  in the `flutter` repository on GitHub.
```